tests/subsys/settings/functional/nvs/boards/native_posix.conf

```diff
@@ -0,0 +1 @@
+CONFIG_SETTINGS_NVS_SECTOR_SIZE_MULT=8192
```

Comment:
Is this a configuration that should always be used for native posix? If so, set it in `boards/posix/native_posix/Kconfig.defconfig` instead?